plugins/modules/s3_bucket_info.py

```diff
@@ -112,8 +112,8 @@
   transform_location:
     description:
       - S3 bucket location for default us-east-1 is normally reported as C(null).
-      - Setting this option to C(true) will return C(us-east-1) instead.
-      - Affects only queries with I(bucket_facts=true) and I(bucket_location=true).
+      - Setting this option to V(true) will return C(us-east-1) instead.
```

Comment:
```suggestion
      - Setting this option to V(true) will return RV(us-east-1) instead.
```

Reply:
us-east-1 is not a returned value really. If I use RV(), the documentation building step will fail. I use V() instead.